src/index.css

```diff
@@ -23,4 +23,43 @@ body {
     sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  background: rgb(176, 207, 176);
+}
+
+main {
+  min-height: 100vh;
+  display: flex;
+  justify-content: center;
+  margin: 0 auto;
+}
+
+.calculator {
+  display: grid;
+  grid-template-columns: repeat(4, 6rem);
+  grid-template-rows: minmax(7rem, auto) repeat(5, 6rem);
+  margin-top: 20px;
+}
+
+.btn {
+  grid-column: span 2;
+}
+
+.header {
+  grid-column: 1 / -1;
+  background: #555;
+  display: flex;
+  align-items: flex-end;
+  justify-content: space-around;
+  flex-direction: column;
+}
+
+.calc {
+  color: #fff;
+  padding-right: 20px;
+  font-size: 2rem;
+}
+
+button {
+  font-size: 1.5rem;
+  cursor: pointer;
 }
```

Comment:
- [ ] The design of your calculator is not identical to that of the sample template provided. Kindly match the design in terms of **_colour_**, and **_allocation of spaces for each button_** in the calculator. For example, the **_Zero (0)_** button spans two grid spaces but yours spans one while the **_Equal (=)_** button spans one grid space but yours spans two spaces. 💻 🖱️ 

### Your Design

![image](https://github.com/Umuhan/calculator-app/assets/77106962/25c2426b-070e-4cbe-9c59-522384e309b4)

### Sample Design

![image](https://github.com/Umuhan/calculator-app/assets/77106962/c73cb42f-3a98-4c06-bc4c-216f40694b68)

Reply:
thanks done.